packages/framework/src/Model/Product/Elasticsearch/ProductExportRepository.php

```diff
@@ -80,6 +113,134 @@ public function __construct(
         $this->domain = $domain;
         $this->productVisibilityRepository = $productVisibilityRepository;
         $this->friendlyUrlFacade = $friendlyUrlFacade;
+        $this->categoryFacade = $categoryFacade;
+        $this->productAccessoryFacade = $productAccessoryFacade;
+        $this->pricingGroupSettingFacade = $pricingGroupSettingFacade;
+        $this->brandCachedFacade = $brandCachedFacade;
+    }
```

Comment:
_random line_ 

I think it is worth noting that the time for export will be extended (@sslt as PO should be aware of that). 
Tested on 7000 products (performance data) and time was 1m 45s -> 2m 40s (approx)

Reply:
Told you...

Reply:
please @sslt, resolve this comment during business validation

Reply:
Thank you, I'm aware the speed penalty and reasons behind it, however there is not reasonable way to get around the problem, please proceed.